tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -341,6 +341,21 @@ fn build_nsis_app_installer(
   }
 
   let mut handlebars = Handlebars::new();
+  handlebars.register_escape_fn(|s| {
+    let mut output = String::new();
+    for c in s.chars() {
+      match c {
+        '\"' => output.push_str("$\\\""),
+        '$' => output.push_str("$$"),
+        '`' => output.push_str("$\\`"),
```

Comment:
I couldn't find any documentation related to escaping ``` ` ``` so I am fine with this one until someone else reports it is an issue.